pkg/agent/flannel/setup.go

```diff
@@ -216,7 +231,16 @@ func createFlannelConf(nodeConfig *config.Node) error {
 		backendConf = strings.ReplaceAll(wireguardBackend, "%flannelConfDir%", filepath.Dir(nodeConfig.FlannelConfFile))
 		logrus.Warnf("The wireguard backend is deprecated and will be removed in k3s v1.26, please switch to wireguard-native. Check our docs for information about how to migrate")
 	case config.FlannelBackendWireguardNative:
-		backendConf = wireguardNativeBackend
+		mode, ok := backendOptions["mode"]
+		if !ok {
+			mode = "separate"
+		}
+		keepalive, ok := backendOptions["persistent-keepalive"]
```

Comment:
what about

```suggestion
		keepalive, ok := backendOptions["keepalive-interval"]
```

Reply:
fwiw the reason i went for `persistent-keepalive` is because the `wg` command-line tool also uses that naming, so it seemed like a good color to pick. But i don't really mind either naming

Reply:
ah interesting. I think it makes the most sense to use what the flannel config calls it, since we're configuring flannel here and not WG directly.

Reply:
The flannel config calls it `PersistentKeepaliveInterval` and `Mode` ; Would you prefer to call them that (with that capitalization)  rather then respectively `persistent-keepalive` and `mode`

Reply:
Yeah, that'd probably be good, just for future consistency in case we make additional parameters configurable on other backends.